sdk/maps/maps-timezone-rest/README.md

```diff
@@ -0,0 +1,251 @@
+
+const credential = new DefaultAzureCredential();
+const client = MapsTimezone(credential);
+```
+
+#### Using a Subscription Key Credential
+
+You can authenticate with your Azure Maps Subscription Key. Please install the `@azure/core-auth` package:
+
+```bash
+npm install @azure/core-auth
+```
+
+```javascript
+const MapsTimezone = require("@azure-rest/maps-timezone").default;
+const { AzureKeyCredential } = require("@azure/core-auth");
+
+const credential = new AzureKeyCredential("<subscription-key>");
+const client = MapsTimezone(credential);
+```
+
+#### Using a Shared Access Signature (SAS) Token Credential
+
+Shared access signature (SAS) tokens are authentication tokens created using the JSON Web token (JWT) format and are cryptographically signed to prove authentication for an application to the Azure Maps REST API.
+
+You can get the SAS token using [`AzureMapsManagementClient.accounts.listSas`](https://learn.microsoft.com/javascript/api/%40azure/arm-maps/accounts?view=azure-node-latest#@azure-arm-maps-accounts-listsas) from the `@azure/arm-maps` package. Please follow the section [Create and authenticate an `AzureMapsManagementClient`](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/maps/arm-maps#create-and-authenticate-an-azuremapsmanagementclient) to set up first.
+
+Second, follow [Managed identities for Azure Maps](https://techcommunity.microsoft.com/t5/azure-maps-blog/managed-identities-for-azure-maps/ba-p/3666312) to create a managed identity for your Azure Maps account. Copy the principal ID (object ID) of the managed identity.
+
+Finally, you can use the SAS token to authenticate the client:
+
+```javascript
+const MapsTimezone = require("@azure-rest/maps-timezone").default;
+const { AzureSASCredential } = require("@azure/core-auth");
+const { DefaultAzureCredential } = require("@azure/identity");
+const { AzureMapsManagementClient } = require("@azure/arm-maps");
+
+const subscriptionId = "<subscription ID of the map account>";
+const resourceGroupName = "<resource group name of the map account>";
+const accountName = "<name of the map account>";
+const mapsAccountSasParameters = {
+  start: "<start time in ISO format>", // e.g., "2023-11-24T03:51:53.161Z"
+  expiry: "<expiry time in ISO format>", // maximum value to start + 1 day
+  maxRatePerSecond: 500,
+  principalId: "<principal ID (object ID) of the managed identity>",
+  signingKey: "primaryKey",
+};
+const credential = new DefaultAzureCredential();
+const managementClient = new AzureMapsManagementClient(credential, subscriptionId);
+const { accountSasToken } = await managementClient.accounts.listSas(
+  resourceGroupName,
+  accountName,
+  mapsAccountSasParameters,
+);
+if (accountSasToken === undefined) {
+  throw new Error("No accountSasToken was found for the Maps Account.");
+}
+const sasCredential = new AzureSASCredential(accountSasToken);
+const client = MapsTimezone(sasCredential);
+```
+
+## Key concepts
+
+### MapsTimezoneClient
```

Comment:
I'm not sure SDK board will ask us to change it to `MapsTimeZoneClinet` or not. We should ask them beforehand?

Reply:
@Lakicar95 , have you confirmed?